File: Esquenta/Entities/Comanda.cs
using Esquenta.Entities.Interfaces;

namespace Esquenta.Entities
{
    public class Comanda : IBaseEntity
    {
        public virtual int Id { get; protected set; }
        public virtual bool EmAberto { get; set; }
        public virtual string Nome { get; set; }
        public virtual string CodigoBarras { get; set; }
    }
}

File: V2-Node/esquenta.v2/server/models/comanda.js
'use strict'
module.exports = (sequelize, DataTypes) => {
  var comanda = sequelize.define('comanda', {
    nome: DataTypes.STRING,
    codigoBarras: DataTypes.STRING,
    emAberto: DataTypes.BOOLEAN
  }, {})
  comanda.associate = function (models) {
    comanda.hasMany(models.venda, {
      foreignKey: 'comandaId',
      as: 'Vendas'
    })
  }
  return comanda
}
